.gitignore

```diff
@@ -3,9 +3,13 @@ target/
 # vim
 *.sw?
 
+
+.js
+.jvm
+.native
 # Ignore [ce]tags files
 tags
-
+null
```

Comment:
I'm not experiencing the creation of any `null` file in the repo when compiling it or using it, so I think it can be a problem you might be experiencing locally.

There are two solutions here:
1) you can try to find out what's causing it and solve the issue
2) you can add this exclusion in your `.gitignore_global` as you're the only one experiencing it

Reply:
Thank you! I'm going to try the latter as I already have spend a ton of time in the first one 😭